src/main/java/com/google/devtools/build/lib/skyframe/PlatformLookupUtil.java

```diff
@@ -66,6 +85,53 @@
     return platforms;
   }
 
+  /** Validate that all keys are for actual platform targets. */
+  private static void validatePlatformKeys(
+      ImmutableList<ConfiguredTargetKey> platformKeys, Environment env)
+      throws InterruptedException, InvalidPlatformException {
+    // Load the packages. This should already be in Skyframe and thus not require a restart.
+    Map<ConfiguredTargetKey, PackageIdentifier> targetsToPackageIdentifiers =
```

Comment:
This map only seems to be used to create another collection out of its values. Consider merging the two stream operations and/or making this a set of package identifiers.

Reply:
Done.